http-clients/src/main/java/com/palantir/remoting3/clients/ClientConfigurations.java

```diff
@@ -40,6 +40,7 @@
     private static final Duration DEFAULT_CONNECT_TIMEOUT = Duration.ofSeconds(10);
     private static final Duration DEFAULT_READ_TIMEOUT = Duration.ofMinutes(10);
     private static final Duration DEFAULT_WRITE_TIMEOUT = Duration.ofMinutes(10);
+    private static final Duration DEFAULT_BACKOFF_SLOT_SIZE = Duration.ofMillis(500);
```

Comment:
meh, sure - if we were to change this value, how 'breaking' a change would this be? Think likely will want to tweak this as we learn more...

Reply:
Well, we have to set some default. How we are going to change it over time ... I don't know. It could be considered a breaking change, but users can always opt to give an explicit value.